yaml_checker/README.md

```diff
@@ -0,0 +1,111 @@
+# YAML Checker
+
+An internal CLI util for formatting and validating YAML files. This project
+relies on Pydantic and Ruamel libraries.
```

Comment:
I guess the second line is not necessary, since there is a `requirement.txt` which lists those.